### ISSUE TITLE
Regarding TIBCO 6.5.1 HF3 maven plugin support

### ISSUE THREAD

**Issue by kiran8rt** [TIBCOSoftware/bw6-plugin-maven#488]
Hi,

We recently implemented one of our application using TIBCO Business works 6.5.1.

We are using TIBCO maven plugin for deployments of our TIBCO BW6 applications.

Recently we started facing a deployment related issue on 6.5.1 for which we had raised a ticket on TIBCO support. below is the link.

https://support.tibco.com/s/case/5002L00001B8ImFQAV/deployment-issue-on-a-single-appnode-with-same-http-connection

According to tibco support they have resolved the issue in HF3 but the issue is that their is no compatible maven plugin for the same.

We tried installing latest maven plugin 2.7.1 but it also does not support HF3 for BW6.5.1. Can you please check this.

Thanks in advance.

**Comment by vpawar-ai** [comment]
@kiran8rt Please share the sample BW application(to reproduce the issue), maven command and maven debug logs.

**Comment by kiran8rt** [comment]
Hi, 

The issue is not with the maven plugin. The issue was with TIBCO 6.5.1. so they have resolved that product issue in Hotfix 3. But we do not have a compatible maven plugin version. 

Thanks

**Comment by vpawar-ai** [comment]
@kiran8rt Please use version 2.3.1. Please reopen this issue or create new github issue if you face any problems. Thanks.

**Comment by kiran8rt** [comment]
Hi, We are currently using 2.3.1 with TIBCO 6.5.1 but we faced maven deployment errors when we installed HF3.

**Comment by vpawar-ai** [comment]
Please share the error details. Use -X flag while running the maven command to print debug output and share the logs. Thanks.

**Comment by kiran8rt** [comment]
Hi,

Please find attached file for the logs. 
Just an information we have multi version environments that is both 6.5.1 and 6.6 is installed on different servers, so using an old version of maven plugin is not an option. 

Please let me know if anything else is required.

Thanks,
Kiran[
[TIB6UATCH-BW6BUILDDEPLOYUATCHEXJ-JOB1-35.log.txt](https://github.com/TIBCOSoftware/bw6-plugin-maven/files/5089892/TIB6UATCH-BW6BUILDDEPLOYUATCHEXJ-JOB1-35.log.txt)
](url)

**Comment by vpawar-ai** [comment]
Hi @kiran8rt Thanks for sharing the logs. I see couple of issues -

- Since you are using 6.5.1 and 6.6 both, it is recommended to use latest maven plugin version 2.7.0.

- Based on the exception in the logs, it has failed while uploading EAR.
```
build	19-Jun-2020 12:31:59	[INFO] Uploading the Archive file -> bw.mc-exp-internal-ch-wms-out-pocreate.application_1.0.0-SNAPSHOT.ear
build	19-Jun-2020 12:32:00	[ERROR] 
build	19-Jun-2020 12:32:00	com.tibco.bw.maven.plugin.admin.client.ClientException: null: TIBCO-BW-ADMIN-500000: null
build	19-Jun-2020 12:32:00	    at com.tibco.bw.maven.plugin.admin.client.RemoteDeployer.uploadArchive (RemoteDeployer.java:381)
build	19-Jun-2020 12:32:00	    at com.tibco.bw.maven.plugin.admin.client.RemoteDeployer.addAndDeployApplication (RemoteDeployer.java:243)
```

- You are using "bw6-psg-maven-plugin" version 2.6.0. It is not the maven plugin maintained by this github repository. 
```
build	19-Jun-2020 12:31:43	[INFO] --- bw6-psg-maven-plugin:2.6.0:bwdeploy (default-bwdeploy) @ bw.mc-exp-internal-ch-wms-out-pocreate.application ---
```
- Please ensure that the plugin "bw6-maven-plugin" and version 2.7.0 is used across all the pom.xml files in your project.

**Comment by kiran8rt** [comment]
Hi,

Does this 2.7.0 version will support both 6.5.1 with all hot fix including HF3 and also 6.6 versions. Kindly confirm. Thank you in advance.
